### ISSUE TITLE
delete-outer-* doesn't match vim's behavior

### ISSUE THREAD

**Issue by TheBB** [emacs-evil/evil#722]
Originally reported by: **Nick Felix (Bitbucket: [Dickby](https://bitbucket.org/Dickby), GitHub: [Dickby](https://github.com/Dickby))**

----------------------------------------

with the cursor at `|` pressing `d a B` :
```
#!C
pre;
{
    i|n;
{
post;
```
in evil-mode leaves:
```
#!C
pre;
|
post;
```
but in vim:
```
#!C
pre;
|post;
```

----------------------------------------
- Bitbucket: https://bitbucket.org/lyro/evil/issue/722

**Comment by neeasade** [comment]
This is not an actual issue and can be closed

**Comment by Dickby** [comment]
@neeasade, why do you think it isn't?

**Comment by neeasade** [comment]
@Dickby Actually the bug is valid -- I misread the comments on this issue earlier, and was able to reproduce. Apologies.

**Comment by dieggsy** [comment]
huh - I think I also misread

**Comment by tomdl89** [comment]
I can't reproduce this. Maybe it was fixed in the meantime. Feel free to reopen if you can explain how to repro.

**Comment by neeasade** [comment]
There's a typo in the original issue I see. Should be:

```
#!C
pre;
{
    i|n;
}
post;
```

(where [n] is just used to show the cursor over n)

with vim 8.0, launched with `vim -u NONE`, daB results in:

```
#!C
pre;
|post;
```

while in evil, I get: 

```
#!C
pre;
|
post;
```

I'm not too invested in this issue, idk why I was here in 2017 lol

**Comment by tomdl89** [comment]
@neeasade thanks - I should have said, I guessed that was a typo, but still couldn't repro. But I wasn't testing in a vanilla setup. I'll take another look

**Comment by tomdl89** [comment]
Nope, still can't repro. Only difference with vim is that `evil-start-of-line` is not respected by evil for this, whereas startofline is in vim (although this is pretty niche, as it's not even documented for vim).

**Comment by tomdl89** [comment]
It was fixed in #1578
